sprints/dashboard/views.py

```diff
@@ -100,12 +99,12 @@ def is_forbidden_to_end_sprint(board_id: int, acquire_lock: bool = False) -> Opt
         )) or (not acquire_lock and cache.get(f'{settings.CACHE_SPRINT_END_LOCK}{board_id}', False)):
             return "The completion task is already running or hasn't been completed successfully."
 
-        return None
+        return True
```

Comment:
I think it would be better to make this a tuple. The first value can be the True/False and the second can be the reason. That way the response is always consistent. Ideally it might actually be better to split out the true/false from the code that gives the reason why. 

Currently someone else might check this the result of this function as true/false and always get a truthy value.